course_info/icommons.py

```diff
@@ -92,6 +108,31 @@ def _get_schools(self, school_id=None, **kwargs):
             raise ICommonsApiValidationError(str(e))
         return rv
 
+    def _course_info_instructor_list(self, course_instance_id=None, **kwargs):
+
+        # use kwargs to append 'people' to the resource url and to get course people data
+        kwargs['append_to_path'] = 'people'
+
+        instructors = None
+        rv = self._get_resource('course_instances', course_instance_id, **kwargs)
```

Comment:
if `course_instance_id` is indeed `None`, we're just letting `_get_resource_by_url()` eventually fail (because `path += '{}/'.format(id_)` will interpolate `None/`?